st2common/st2common/models/api/pack.py

```diff
@@ -100,6 +100,22 @@ class PackAPI(BaseAPI):
                                '">=1.8.0, <2.2.0"',
                 'pattern': ST2_VERSION_REGEX,
             },
+            'python_versions': {
+                'type': 'array',
+                'description': ('Major Python versiosn supported by this pack. E.g. '
+                                '"2" for Python 2.7.x and "3" for Python 3.6.x'),
```

Comment:
This will change over time, is it wise to put the minor versions in here?  I could see this getting stale.

Reply:
As mentioned here (https://github.com/StackStorm/st2/pull/4474#issuecomment-452314335), the minor and patch version itself are dictated by the StackStorm platform and the packages we ship.

We could also store it here, but I don't think it would serve any good purpose - the information itself would be duplicated and provide no additional value, just potential confusion.

The idea is if you are using official StackStorm packages under Python 2 and pack metadata says it works under Python 2, it should work. Same goes for Python 3.

Reply:
My point was that the string on line 106 will become incorrect over time.